tool_path_planner/include/tool_path_planner/halfedge_edge_generator.h

```diff
@@ -66,6 +66,9 @@ class HalfedgeEdgeGenerator : public PathGenerator
 
     /**@brief point distance parameter used in conjunction with the spacing method */
     double point_dist = 0.01;
+
+    /** @brief maximum segment length */
+    double max_segment_length = 1.0;
```

Comment:
What is this parameter used for in the Plane Slicer?

Reply:
This is in the halfedge edge generator header, which is used for generating edge paths. I assume based on the description that it is the max allows edge segment size. @drchrislewis Is this correct?

Reply:
Yes, this controls the maximum length of an edge segment. There are alternate parameters and methods for subdividing edges that may result in better performance, but each had corner cases that were more complex, so I went with the simplest method to get us going. For example, since exterior edges always start and end at the same point, we could simply sub-divide by some number whenever there was a circuit. It would be really nice to find the best places to subdivide rather than do it evenly. Again these are complex. One could also classify edges and horizontal-left, hor-rt, ver-up and vert-down. and break them up into 4. Once again this is complex to implement, but may be necessary for some applications/customers.